sql/branch.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+    ALTER TABLE participant ADD COLUMN braintree_id text NOT NULL DEFAULT '';
```

Comment:
We have a `braintree_customer_id` field already, is that what you were looking for?